.github/workflows/build-scan-push.yaml

```diff
@@ -91,14 +91,15 @@ jobs:
           labels: |
             org.opencontainers.image.revision=${{ github.event.pull_request.head.sha || github.event.after || github.event.release.tag_name }}
             org.opencontainers.image.source=https://github.com/${{ github.repository }}
-          cache-from: type=gha
-          cache-to: type=gha,mode=max
+#          cache-from: type=gha
+#          cache-to: type=gha,mode=max
```

Comment:
@gtardif The issue with the license key was that the stage was cached, now it works fine:

![image](https://user-images.githubusercontent.com/15997951/191275664-ab73b957-9004-4a1d-9a9a-6a65354a1638.png)